app/views/wikis/destroy.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Wikis#destroy</h1>
```

Comment:
Good point. Think it was generated when I used the Rails Generator and guess I didn't spot it. Will delete it in my current branch and merge.